nixos/modules/services/monitoring/librenms.nix

```diff
@@ -14,6 +14,7 @@ let
     log_errors = on
     post_max_size = 100M
     upload_max_filesize = 100M
+    memory_limit = ${toString cfg.settings.php_memory_limit}M
```

Comment:
```suggestion
    memory_limit = ${cfg.settings.php_memory_limit}
```

Reply:
Same as above: the megabytes are hardcoded in LibreNMS.